### PR TITLE
Port 52516

### PR DESCRIPTION
**Pull request by msteed**
### What does this PR do?

Port #52516 to `master`

### CODE REVIEW

File: tests/unit/test_minion.py [hunk 1]
```diff
@@ -367,3 +368,53 @@ def test_sock_path_len(self):
...
+        patches = [
+            patch('salt.transport.client.AsyncPubChannel.factory', return_value=_mock),
+            patch('salt.minion.Minion.tune_in', return_value=None),
+            patch('salt.minion.Minion._post_master_init', return_value=future_stub),
+            patch('salt.utils.network.dns_check', new=dns_check),
+        ]
+        for _patch in patches:
+            _patch.start()
+
+        mock_opts = salt.config.DEFAULT_MINION_OPTS.copy()
```
Comment (s0undt3ch) [inline tests/unit/test_minion.py]: 💯

**Review by s0undt3ch — DISMISSED** [review]

**Comment by dwoz** [comment]
@msteed We need to resolve the conflicts and linter errors.

**Comment by msteed** [comment]
Hi @dwoz, sorry about this but I am consumed with other work and won't be getting to this any time soon. I'm okay just closing it if the cost of keeping it limping along is more than it is worth.

**Comment by garethgreenaway** [comment]
Conflict and linter issues resolved.

**Comment by waynew** [comment]
I think this just needs a rebase on master

**Comment by dwoz** [comment]
Closing this due to inactivity. Anyone should feel free to re-open it if they want to see it through to the end in one release cycle.

**Pull request closed without merging**